src/routes/api/users/getById.js

```diff
@@ -1,16 +1,44 @@
 const logger = require("../../../logger");
+const validator = require("validator");
+const { PrismaClientKnownRequestError } = require("@prisma/client").Prisma;
+const { get } = require("../../../model/user");
```

Comment:
suggestion: Just so its clearer what we're `get`ing

1/2

```suggestion
const user = require("../../../model/user");
```